generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -279,6 +282,18 @@ webpack-test:
         paths:
             - build/test-results
         expire_in: 1 day
+<%_ if (cicdIntegrations.includes('sonar')) { _%>
```

Comment:
we should handle the case !skipServer isnt'it ?
We already have a sonar plugin in both maven and gradle config files. We need to check if server is not skipped so we have to call wether maven or gradle sonar stage.